pkgs/development/compilers/z88dk/Makefile.patch

```diff
@@ -1,5 +1,5 @@
---- /nix/store/lqr8s3yi8gn98gf3xybrmxg2dj1d4li2-source/Makefile	1970-01-01 01:00:01.000000000 +0100
-+++ lqr8s3yi8gn98gf3xybrmxg2dj1d4li2-source/Makefile	2018-02-16 13:28:44.979280387 +0100
+--- joachifm-rtfm/Makefile	1970-01-01 01:00:01.000000000 +0100
```

Comment:
Classy ...

Reply:
the origin of diff is not a problem, how can i spend my time fixing real stuff and doing good job ... and my sense of humor is like that :) i would prefer to keep the first one since it doesn't matter ...

Reply:
i should not have to edit a patch file generated by diff -Naur , and i would prefer fixing upstream later.